content/docs/tools-ui/ui-dev-messages.md

```diff
@@ -0,0 +1,537 @@
+
+
+## Standard Icons
+
+The standard icons built into O3DE are the **Error**, **Warning**, **Success**, and **Information** icons. They communicate different severity levels of information to users.
+
+These icons are displayed on various O3DE UI systems such as *Toast Messages*, *Dialogs*, and *Message Boxes*. They are also commonly displayed within *Event Logs*, *Console Logs*, and *Component Cards*.
+
+#### Error / Failure Icon
+
+![O3DE Standard Error or Failure Icon](/images/tools-ui/overview/standard-icons/error-or-failure.png)
+
+Indicates that a process has failed, or there is an error that requires immediate user action---that is, the user might be critically impacted, or in a potentially irreversible state. Errors or failures may be an irregularity in the engine, a malfunctioning feature, or a request from the user to confirm a destructive action.
+
+*Used for: errors, failure, failed processes, emergencies, urgent alerts*
+
+In O3DE, we define two different terms to describe an issue: *Error* and *Failure*. An *Error* is the deviation from the actual and expected output. A *Failure* is the inability of the system to perform a required function.
+
+#### Warning Icon
+
+![O3DE Standard Warning Icon](/images/tools-ui/overview/standard-icons/warning.png)
+
+Indicates a situation that requires corrective action from the user to prevent a critical fault, or a situation in which a non-critical error has occurred. Warnings don't indicate immediate impact on the user, but they suggest that the user performs a preventive action for a streamlined workflow.
+
+*Used for: warnings, unavailability, caution, prevention, instability*
+
+#### Success Icon
+
+![O3DE Standard Success Icon](/images/tools-ui/overview/standard-icons/success.png)
+
+Indicates successfully finished processes and tasks, or implies that no issues are present within the system.
+There is a need to provide feedback to a certain action, or that no immediate user action is required.
+
+*Used for: success, completion, stability, active status, verification, progress indicator*
+
+#### Information Icon
+
+![O3DE Standard Information Icon](/images/tools-ui/overview/standard-icons/information.png)
+
+Indicates additional (non-critical) information that doesn't require action from the user. This may be used to inform the user that something is ready to view, such as for system feedback or to signify changes since the last interaction.
+
+*Used for: information, guidance, exceptions*
+
+
+## Message Colors
+
+In messages, semantic colors indicate the purpose of a message, which users can quickly recognize by the color. For instance, the color green has positive connotations, so it's used to communicate success or confirmation.
+
+- Error: `#FA2727`
+- Warning: `#FFAA22`
+- Success: `#58BC61`
+- Information: `#1E70EB`
+- Information icon: `#FFFFFF`
+
+![O3DE Message Colors](/images/tools-ui/overview/colors.png)
```

Comment:
Previous text:
- [Line 84]
	> We use semantic colors to indicate purpose; these help users convey specific messages. For instance, green has positive connotations, so it's used to communicate success or confirmation.

Comments: 
- Added "In messages" to provide context where you are using the semantic colors. 
- Changed "indicate purpose" to "indicate the purpose of a message" to clarify whose "purpose" we are talking about. 
- Rephrased "these help users convey specific messages" to "...the purpose of a message, which users can quickly recognize by the color". It's important to be clear about the "user" here because the user can be the one developing the UI message, or the one reading it -- let's use "developer" and "user", respectively, to differentiate. I think "help \[developers\] convey specific messages" is repetitive of "to indicate a purpose" earlier in the sentence. So instead, we can write about the purpose from the user's perspective, which is to recognize the message by the color.
- Added a list of the color hex codes, so developers can copy and paste.